cypress/e2e/signUp/signUp.spec.cy.js

```diff
@@ -0,0 +1,9 @@
+describe('Login', () => {
+    it('SignUp with valid number', () => {
+      cy.visit('https://staging.circlepay.ai/')
+      cy.get('.mat-input-element').type('+923001231231').should('have.value','3001231231')
```

Comment:
@AmnaShahid96 same comment about the phone number, save it to environment variable.